README.md

```diff
@@ -50,10 +50,11 @@ Returns source map converter from given `filename` by parsing `//# sourceMapping
 `filename` must point to a file that is found inside the `mapFileDir`. Most tools store this file right next to the
 generated file, i.e. the one containing the source map.
 
-### fromSource(source)
+### fromSource(source[, largeSource])
 
-Finds last sourcemap comment in file and returns source map converter or returns null if no source map comment was
-found.
+Finds last sourcemap comment in file and returns source map converter or returns null if no source map comment was found.
+
+`largeSource` is optional and useful if you're running into "call stack size exceeded" errors. If set to `true`, an algorithm that does not use regex is applied. It faster but potentially less accurate because it might find comments that aren't sourcemaps.
```

Comment:
Minor changes:

If `largeSource` is set to `true` an algorithm that does not use regex is used to find the source map.This is useful if you're running into "call stack size exceeded" errors and is also faster.   
However it is less accurate and may match content that isn't a source map comment.